### PR TITLE
Feature/cypress testing

### LINKED ISSUE #14: Write Cypress user flow testing
- [ ] User should be able to see home page 
- [ ] User should see on page title and instructions
- [ ] User should see

### PR DESCRIPTION
**Pull request by maeduphorne**
# Thank you for pushing your code!
## What (if any) features are you implementing?
- Implement basic cypress testing for first user flow  on the main page

## What (if anything) did you refactor?
- N/A

## Were there any issues that arose?
- No issues!

## Is there anything that you need from your teammate?
- Let me know if there are any tests you'd like to add. We should think about sad paths.

## Any other comments, questions, or concerns?
- Next step is to intercept and stub to test API

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?
- [ ] open index.html
- [ ] console.log()
- [x] dev tools

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

closes #14

### CODE REVIEW

File: cypress/integration/homepage_spec.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    beforeEach(() => {
+        cy.visit('http://localhost:3000/')
+        });
+
+    it('Should have a header with the App\'s name on page load & an image of a map', () => {
+        cy.get('h1').contains('What In The World')
+        cy.get('.worldMapImg').should('be.visible')
+    })
+
+    // should display a drop down form of countries on load
```
Comment (Fakhatova) [inline cypress/integration/homepage_spec.js:11]: awesome commenting out every it block purposes

File: cypress/integration/homepage_spec.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+describe('Main Page Render', () => {
+    beforeEach(() => {
```
Comment (MTCawthray) [inline cypress/integration/homepage_spec.js:2]: Do we want to do an intercept and stub out 1 country to check for in the dropdown?

File: src/components/App/App.tsx [hunk 3]
```diff
@@ -84,7 +84,7 @@ function App() {
               
                 <button onClick={(e) => handleSubmit(e)} 
                   className="dropdown-btn">
-                  <Link to={`/${selectedCountry}`}>
+                  <Link className= "country-submit" to={`/${selectedCountry}`}>
```
Comment (MTCawthray) [inline src/components/App/App.tsx:81]: Good call adding a class name to grab onto for testing

**Review by MTCawthray — APPROVED** [review]
Looking good! I'm thinking we'd want to intercept and stub on page load to control the data that populates into the dropdown menu. This is a great place to jump off from for the rest of our user flows. Also, great job on troubleshooting the missing integration file situation!

**Pull request merged**